python/ray/data/_internal/datasource/webdataset_datasource.py

```diff
@@ -342,6 +344,8 @@ def _read_stream(self, stream: "pyarrow.NativeFile", path: str):
         Yields:
             List[Dict[str, Any]]: List of sample (list of length 1).
         """
+        import json
```

Comment:
I think you can just import at top-level

Reply:
ok